planning/autoware_velocity_smoother/config/default_velocity_smoother.param.yaml

```
@@ -9,7 +9,10 @@
 
     # ---- curve parameters  ----
     # - common parameters -
-    curvature_calculation_distance: 5.0         # distance of points while curvature is calculating for the steer rate and lateral acceleration limit [m]
+    curvature_calculation_distance: 5.0   # distance of points while curvature is calculating for the steer rate and lateral acceleration limit [m]
+    ego_nearest_dist_threshold: 0.3       # threshold to find the nearest point on the trajectory [m]
+    ego_nearest_yaw_threshold: 1.046      # threshold to find the nearest point on the trajectory [rad]
+    algorithm_type: "JerkFiltered"       # algorithm type for velocity smoother. "JerkFiltered" or "L2" or "Linf" or "Analytical"
```

Could you change this to the following code so that [the argument](https://github.com/autowarefoundation/autoware.universe/blob/2a2d037820bdda445675061168a4d1d8378a72dc/planning/autoware_velocity_smoother/launch/velocity_smoother.launch.xml#L12) passed to the launch file is used instead?

```suggestion
    algorithm_type: $(var velocity_smoother_type)       # algorithm type for velocity smoother. "JerkFiltered" or "L2" or "Linf" or "Analytical"
```